site/content/docs/setup/examples/postgres-docker-compose.yml

```diff
@@ -14,8 +14,8 @@ services:
       - backend_data:/data
     environment:
       # !!! REPLACE These !!!
-      JWT_SECRET: "bXktc2VjcmV0"
-      CORS_ORIGINS: "*"
+      JWT_SECRET: "qDSFr/3Mq9zX+CVYtlSjpmJyYMNXYGBNLDSu51RAuNQ="
```

Comment:
can we keep the jwt secret as it was?

Reply:
I'll do anyway you like!

I do think it often helps to show users what it *should* look like, but then again that means lazy folks will use what you give them. I can revert that change and update the PR

Reply:
I think it would be best to be the original. It is still a valid secret.

Reply:
Reverted the JWT change!

Reply:
Thanks, will merge now!